src/Test.FunctionalTests.Sarif/TestData/Multitool/BaselineOption/Inputs/TEST1002.Baseline.sarif

```diff
@@ -24,21 +24,31 @@
               "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
             },
             {
-              "id": "SARIF2003",
-              "name": "ProvideVersionControlProvenance",
+              "id": "SARIF2002",
+              "name": "ProvideMessageArguments",
               "fullDescription": {
-                "text": "Provide 'versionControlProvenance' to record which version of the code was analyzed, and to enable paths to be expressed relative to the root of the repository."
+                "text": "In result messages, use the 'message.id' and 'message.arguments' properties rather than 'message.text'. This has several advantages. If 'text' is lengthy, using 'id' and 'arguments' makes the SARIF file smaller. If the rule metadata is stored externally to the SARIF log file, the message text can be improved (for example, by adding more text, clarifying the phrasing, or fixing typos), and the result messages will pick up the improvements the next time it is displayed. Finally, SARIF supports localizing messages into different languages, which is possible if the SARIF file contains 'message.id' and 'message.arguments', but not if it contains 'message.text' directly."
               },
```

Comment:
Remove This has several advantages.

Reply:
Everything in the TestData folder is just that, so it's not user-facing and definitely not worth the time. :)